plugin/evm/vm_test.go

```diff
@@ -79,19 +79,30 @@ func init() {
 	genesisBalance = new(big.Int).Mul(big.NewInt(testMinGasPrice), big.NewInt(21000*1000))
 }
 
-// BuildGenesisTest returns the genesis bytes for Subnet EVM VM to be used in testing
-func BuildGenesisTestWithAlloc(t *testing.T, genesisJSON string) []byte {
-	ss := CreateStaticService()
-
+func fundGenesis(genesisJSON string) (string, error) {
```

Comment:
Since we are calling this everywhere to fund the two addresses, can we instead just modify the balance on the genesis string and use that?

Reply:
done